vcd/resource_vcd_vapp_vm.go

```diff
@@ -86,14 +87,66 @@ func resourceVcdVAppVm() *schema.Resource {
 				Default:  true,
 			},
 			"network_href": &schema.Schema{
-				Type:     schema.TypeString,
-				Optional: true,
+				Type:       schema.TypeString,
+				Optional:   true,
+				Deprecated: "In favor of networks",
+			},
+			"networks": {
+				ConflictsWith: []string{"ip", "network_name"},
+				ForceNew:      true,
+				Optional:      true,
+				Type:          schema.TypeList,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"orgnetwork": {
+							ForceNew: true,
+							Required: true,
+							Type:     schema.TypeString,
+						},
+						"ip": {
+							Computed:         true,
+							DiffSuppressFunc: suppressIfIpIsOneOf(),
+							ForceNew:         true,
+							Optional:         true,
+							Type:             schema.TypeString,
+						},
+						"ip_address_allocation_mode": {
+							Default:      "POOL",
+							ForceNew:     true,
+							Optional:     true,
+							Type:         schema.TypeString,
+							ValidateFunc: checkIpAddressAllocationMode(),
+						},
+						"is_primary": {
+							Default:  false,
+							ForceNew: true,
+							Optional: true,
+							Type:     schema.TypeBool,
+						},
+						/*
```

Comment:
Nit ;) Maybe it's just me, but these in-line comments between `/*` and `*/` poking out, especially because the paragraph is indented. What about just using `//`?

Reply:
Fixed.